framework/Samplers/Sampler.py

```diff
@@ -498,6 +513,17 @@ def localInitialize(self):
     """
     pass
 
+  def _constantVariables(self):
+    """
+      Method to set the constant variables into the inputInfo dictionary
+      @ In, None
+      @ Out, None
+    """
+    if len(self.constants) > 0:
+      self.inputInfo['SampledVars'  ].update(self.constants)
+      self.inputInfo['SampledVarsPb'].update(dict.fromkeys(self.constants.keys(),1.0))
```

Comment:
Should we add a comment here just to clarify that we're setting the probability weight for constant variables to 1?  It seems obvious, but could avoid confusion later when someone working on something else hits this line.